seqr/views/apis/individual_api_tests.py

```diff
@@ -463,6 +513,310 @@ def test_individuals_table_handler(self):
         response = self.client.post(save_url)
         self.assertEqual(response.status_code, 200)
 
+    @mock.patch('seqr.views.utils.permissions_utils.PM_USER_GROUP', 'project-managers')
+    @mock.patch('seqr.views.utils.pedigree_info_utils.NO_VALIDATE_MANIFEST_PROJECT_CATEGORIES')
+    @mock.patch('seqr.utils.communication_utils.EmailMultiAlternatives')
+    def test_individuals_sample_manifest_table_handler(self, mock_email, mock_no_validate_categories):
+        receive_url = reverse(receive_individuals_table_handler, args=[PROJECT_GUID])
+        self.check_manager_login(receive_url)
+
+        def _send_request_data(data):
+            return self.client.post(receive_url, {'f': SimpleUploadedFile(
+                'sample_manifest.tsv', '\n'.join(['\t'.join(row) for row in data]).encode('utf-8')),
```

Comment:
It can be a class function and shared with other tests.

Reply:
Thats out of scope for this PR, the goal here is to make our tests able to catch an existing bug and any similar bugs going forward. Refactoring the other tests in this file is out of scope